src/SpaceStation.cpp

```diff
@@ -355,6 +367,27 @@ bool SpaceStation::GetDockingClearance(Ship *s, std::string &outMsg)
 			sd.maxOffset = std::max((pPort->maxShipSize / 2 - bboxRad), float(pPort->maxShipSize / 5.0));
 			sd.maxOffset *= sd.maxOffset;
 			outMsg = stringf(Lang::CLEARANCE_GRANTED_BAY_N, formatarg("bay", i + 1));
+			if (this->IsGroundStation()) {
+				Frame *frame = Frame::GetFrame(this->GetFrame());
+				Body *stationPlanet = frame->GetBody();
+				Planet *p = static_cast<Planet *>(stationPlanet);
+
+				double pressure, density;
+				p->GetAtmosphericState(this->GetPositionRelTo(stationPlanet).Length(), &pressure, &density);
+
+				double rsqr = GetPositionRelTo(stationPlanet).LengthSqr();
+				// use g instead of m/(s^2) because it is easier to compare with ship info during gameplay
+				double gravity = (G * stationPlanet->GetMass() / rsqr) / 9.80;
+
+				if (pressure > 0.01) {
+					outMsg += " " + stringf(Lang::STATION_GRAV_PRESS, formatarg("grav", gravity, "f.2"), formatarg("press", pressure, "f.2"));
+				} else {
+					outMsg += " " + stringf(Lang::STATION_GRAV, formatarg("grav", gravity, "f.2"));
+				}
+			}
+			if (GetNearbyTraffic(50000) > 0) { //50km
```

Comment:
The nearby traffic variable in lua side is 20km, and here 50km? Ignore this comment if this is correct, I'm not doing a detailed reading of the code.

Reply:
I don't quite remember if I did that 20km on purpose... They should probably be equal.

Reply:
Is it possible to share the variable, so it's imported across c++/lua side? Or maybe that's overkill?

Reply:
It looks like that requires some brain effort, so I'm not figuring it out today. I just changed the value to 50km for now.

Reply:
The orbital SOI is 20km, maybe that's why?